projects/templates/plop-templates/spectrum-component.css.hbs

```diff
@@ -0,0 +1,13 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
```

Comment:
We can remove this from the template I think; no reason to add it for new components since it would theoretically be manually written.